modal/functions.py

```diff
@@ -844,9 +844,15 @@ def __init__(
 
         self._function_handle = function_handle
 
-        rep = r"Function({self._tag})"
+        rep = f"Function({self._tag})"
```

Comment:
No it creates a regular string, but parses the literal in "raw" mode which treats backslashes as literal backslashes rather than escape sequences (but it is mostly used for regex since that's often the behavior you want to make the regex readable)